docs/overview.md

```diff
@@ -63,24 +63,28 @@ change the existing SQL dialects.
 In the meantime, other query engine support is on the roadmap, including
 [Apache Spark](https://spark.apache.org/), [Apache Flink](https://flink.apache.org/) and others.
 
-### AI asset management (WIP)
+### AI asset management
 
-The goal of Gravitino is to unify the data management in both data and AI assets. The support of AI
-assets like models, features, and others are under development.
+The goal of Gravitino is to unify the data management in both data and AI assets, including raw files, models, etc.
 
 ## Terminology
 
-### The model of Apache Gravitino
-
-![Gravitino Model](assets/metadata-model.png)
+### The metadata object of Apache Gravitino
 
 * **Metalake**: The top-level container for metadata. Typically, one group has one metalake
   to manage all the metadata in it. Each metalake exposes a three-level namespace(catalog.schema.
   table) to organize the data.
 * **Catalog**: A catalog is a collection of metadata from a specific metadata source.
   Each catalog has a related connector to connect to the specific metadata source.
-* **Schema**: A schema is equivalent to a database, Schemas only exist in the specific catalogs
-  that support relational metadata sources, such as Apache Hive, MySQL, PostgreSQL, and others.
+* **Schema**: Schema is the second level namespace to group a collection of metadata, schema can
```

Comment:
third level namespace?

Reply:
We don't include `metalake` into the namespace, so `Schema` is the second level namespace.